cpp/include/cudf/detail/copy_if_else.cuh

```diff
@@ -74,7 +74,7 @@ __launch_bounds__(block_size) __global__
     // update validity
     if (has_nulls) {
       // the final validity mask for this warp
-      int warp_mask = __ballot_sync(0xFFFF'FFFF, opt_value.has_value());
+      int warp_mask = __ballot_sync(0xFFFFFFFF, opt_value.has_value());
```

Comment:
nit:
```suggestion
      int warp_mask = __ballot_sync(0xFFFFFFFFu, opt_value.has_value());
```

Reply:
If we want to move all these over to explicit unsigned I think we should do that in a different PR so we capture existing usages of the literal `0xFFFFFFFF` ( and `0xffffffff` )